packages/lab/stories/metric.stories.tsx

```diff
@@ -247,23 +246,3 @@ export const WithCustomIndicator: ComponentStory<typeof Metric> = () => (
     </Metric>
   </div>
 );
-
-export const WithCustomIndicatorColour: ComponentStory<typeof Metric> = () => (
-  <div
-    style={{
-      display: "flex",
-      justifyContent: "space-between",
-      width: 400,
-    }}
-    className="CustomIndicatorColor"
-  >
-    <Metric direction="up" showIndicator>
-      <MetricHeader subtitle="Total Value" title="Revenue YTD" />
-      <MetricContent value="$801.9B" />
-    </Metric>
-    <Metric direction="down" showIndicator>
-      <MetricHeader subtitle="Total Value" title="Revenue YTD" />
-      <MetricContent value="$801.9B" />
-    </Metric>
-  </div>
-);
```

Comment:
This is backwards compat?

Reply:
It's a custom override since we're not focusing on for V1 it's something that I think could be removed. Happy to add it back once we want to focus on customisability.